recipes/epics-striptool/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "epics-striptool" %}
+{% set version = "2_5_17_0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace('_','.') }}
+
+source:
+  url: https://epics.anl.gov/download/extensions/StripTool{{ version }}.tar.gz
+  sha256: a50e4f411e9bb2426d98fb85e27a1a0e50b6aef6ff715c538b57c83127d87d8b
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - epics-base
+    - epics-extensions
+    - openmotif-dev
+    - xorg-libx11  # [unix]
+    - xorg-libxext  # [unix]
+    - xorg-libxmu  # [unix]
+    - xorg-libxp  # [unix]
+    - xorg-libxt  # [unix]
+    - xorg-libxpm  # [unix]
+  run:
+    - epics-base
+    - openmotif
+    - xorg-libx11  # [unix]
+    - xorg-libxmu  # [unix]
+    - xorg-libxt  # [unix]
+    - xorg-libxpm  # [unix]
+
+test:
+  commands:
+    - test -x ${PREFIX}/epics/bin/${EPICS_HOST_ARCH}/StripTool
+
+about:
+  home: https://epics.anl.gov/extensions/StripTool/index.php
+  license: EPICS
```

Comment:
Is there a license file we can include in the package?

Reply:
The linter failed here silently here :-/

Reply:
Filed a feedstock issue ( https://github.com/conda-forge/epics-striptool-feedstock/issues/1 )

Reply:
I sent an email to the EPICS collaboration to see if a license file can be added to the source.